docker/app/entrypoint.sh

```diff
@@ -46,17 +46,21 @@ T1=`date +%s`
 ASSET_CHECKSUM=$(ASSETS_PREFIX=${ASSETS_PREFIX} ./bin/asset_checksum) # This should return the same hash as the call in asset_compiler.rb
 T2=`date +%s`
 echo "...checksumming took $(expr $T2 - $T1) seconds"
+
+echo "asset.checksum.log"
+cat asset.checksum.log
```

Comment:
Did you intend to leave these in?

Reply:
No. I'll remove them or comment them out.